src/motor_data.h

```diff
@@ -30,6 +30,7 @@ typedef struct {
     float abs_erpm_smooth;
     float last_erpm;
     int8_t erpm_sign;
+    int8_t last_erpm_sign;  // last erpm sign prior to footpads disengaging
```

Comment:
@lukash is probably going to tell you to delete your code comment 😂 because none of the others have them

Reply:
we need more comments, not less

Reply:
I agree, but not everyone does

Reply:
This comment is good, because it's not clear what "last" means in the variable name. It's totally fine this one has a comment and the others don't.

If I was to be extra nitpicky, saying just _"... prior to disengaging"_ would be a bit better.